cookbook/cookbook/src/pages/autorization-page/autorization-page.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+
+export class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userNameValue: '',
+      passwordValue: ''
+    }
+  };
+
+  changeInputUserName = (event) => {
```

Comment:
it could be improved this way: https://reactjs.org/docs/forms.html#handling-multiple-inputs

We did something like this in class